engine/src/main/java/org/terasology/rendering/dag/stateChanges/BindFBO.java

```diff
@@ -37,10 +37,10 @@
 
     public BindFBO(String fboName) {
         this.fboName = fboName;
-        if (fboName.equals(DEFAULT_FRAME_BUFFER_NAME)) {
-            fboId = DEFAULT_FRAME_BUFFER_ID;
-        } else {
+        if (!fboName.equals(DEFAULT_FRAME_BUFFER_NAME)) {
             fboId = frameBuffersManager.getFBO(fboName).fboId;
+        } else {
+            fboId = DEFAULT_FRAME_BUFFER_ID;
```

Comment:
That been said, I just reviewed the rest of this PR and apart from these issues I couldn't find any other problem. So, we are almost there with this one too. :smiley: